src/gt4py/next/program_processors/runners/dace_iterator/utility.py

```diff
@@ -59,8 +59,8 @@ def create_memlet_at(source_identifier: str, index: tuple[str, ...]):
 
 def map_nested_sdfg_symbols(
     parent_sdfg: dace.SDFG, nested_sdfg: dace.SDFG, array_mapping: dict[str, dace.Memlet]
-) -> dict[str, Any]:
-    symbol_mapping: dict[str, Any] = {}
+) -> dict[str, str]:
+    symbol_mapping: dict[str, str] = {}
```

Comment:
This function seems to do the same thing as mapping the symbols of connectivities in `_visit_scan_stencil_closure`. Can it not be used there to avoid code duplication?

Reply:
Right, fixed in next commit.